<?xml version="1.0"?>
<launch>
  <node name="niro_uds" pkg="can_interface" type="niro_uds.py" />
</launch>
